javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java

```diff
@@ -73,6 +77,14 @@ private void alterCommitIdColumnIfNeeded() {
         }
     }
 
+    /**
+     * This method is needed for upgrading TEXT columns to VARCHAR(MAX) since TEXT is deprecated.
+     */
+    private void alterMssqlTextColumns() {
```

Comment:
Yep, I tested upgrading an old schema by committing to javers, I also added in a check for the type of the column before running the upgrade.